tests/test_nitpicking.py

```diff
@@ -217,13 +239,29 @@ def test_code_workspace_file(tmp_path: Path, cloned_template: Path):
         assert (tmp_path / "doodba.other2.code-workspace").is_file()
         with (tmp_path / "doodba.other2.code-workspace").open() as fp:
             workspace_definition = json.load(fp)
-        assert workspace_definition == {
-            "folders": [
-                {"path": "odoo/custom/src/zzz"},
-                {"path": "odoo/custom/src/private"},
-                {"path": "."},
+        assert workspace_definition["folders"] == [
+            {"path": "odoo/custom/src/zzz"},
+            {"path": "odoo/custom/src/private"},
+            {"name": f"doodba.{tmp_path.name}", "path": "."},
+        ]
+        # Firefox debugger configuration
+        firefox_path = "${workspaceRoot:private}/test_module_static/static/"
+        firefox_path_exists = [
+            firefox_path == path_map["path"]
+            for path_map in workspace_definition["launch"]["configurations"][1][
+                "pathMappings"
             ]
-        }
+        ]
+        assert any(firefox_path_exists)
+        # Chrome debugger configuration
+        chrome_path = firefox_path
+        chrome_path_exists = [
+            chrome_path == path_map
+            for path_map in workspace_definition["launch"]["configurations"][2][
+                "pathMapping"
+            ].values()
+        ]
+        assert any(chrome_path_exists)
```

Comment:
Here also it can be better, but you'll know how to get it, based on the comment above.